lib/JsonpMainTemplatePlugin.js

```diff
@@ -36,7 +36,7 @@ JsonpMainTemplatePlugin.prototype.apply = function(mainTemplate) {
 			"var script = document.createElement('script');",
 			"script.type = 'text/javascript';",
 			"script.charset = 'utf-8';",
-			"script.async = true;",
+			"script.async = false;",
```

Comment:
there it doesn't
https://github.com/webpack/webpack/blob/1.0/lib/JsonpMainTemplate.js#L61